diff --git a/node_modules/@crxjs/vite-plugin/dist/index.mjs b/node_modules/@crxjs/vite-plugin/dist/index.mjs
index 5c3f629..a077883 100644
--- a/node_modules/@crxjs/vite-plugin/dist/index.mjs
+++ b/node_modules/@crxjs/vite-plugin/dist/index.mjs
@@ -1882,7 +1882,7 @@ const pluginWebAccessibleResources = () => {
         if (contentScripts.size > 0) {
           const viteManifest = parseJsonAsset(
             bundle,
-            "manifest.json"
+            ".vite/manifest.json"
           );
           const viteFiles = /* @__PURE__ */ new Map();
           for (const [, file] of Object.entries(viteManifest))
